pom.xml

```diff
@@ -259,8 +274,48 @@
         <exclusion><groupId>org.locationtech.geogig</groupId><artifactId>geogig-web-api</artifactId></exclusion>
         <exclusion><groupId>org.locationtech.geogig</groupId><artifactId>geogig-osm</artifactId></exclusion>
         <exclusion><groupId>com.jolbox</groupId><artifactId>bonecp</artifactId></exclusion>
-      </exclusions>
-    </dependency>    
+      </exclusions -->
+    </dependency>
+	<dependency>
+      <groupId>ch.qos.logback</groupId>
+      <artifactId>logback-core</artifactId>
+      <version>${logback.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>ch.qos.logback</groupId>
+      <artifactId>logback-classic</artifactId>
+      <version>${logback.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>org.locationtech.geogig</groupId>
```

Comment:
@afabiani If you change the dependency of gs-geogig to the GeoServer artifact, you may not need to explicitly add these GeoGig dependencies.